docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,521 @@
+
+5. You run Basic UI Web application. You will see the pop-up as below:
+
+**Figure: Addon Pop-up on Resume Event in Basic UI**
+
+![Showing pop-up in Basic UI](media/hello_resume.png)
+
+
+### Loading HTML file from addon
+
+In an addon, you can load your own HTML file. In this section, an example of an HTML file that shows `Hello, I'm an Addon` is explained.
+
+Ensure that all the files related to an addon are located under `shared\res` of your addon project. Therefore, the added HTML file must also be located in the same directory.
+
+![Add a new file](media/new_file.png)
+
+![Generate the HTML file](media/generate_hello_html.png)
+
+Following is an example of `hello.html` HTML file:
+
+```
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, width=device-width">
+</head>
+<body>
+    <h1>Hello, I'm an Addon</h1>
+</body>
+</html>
+```
+
+The HTML file should be loaded in the addon. To do that, you can use `addonapi.window.loadURL()` when the resume event occurs. Then, the `rusume()` will look like:
+
+```
+resume(winId) {
+    console.log('TmplDefault.resume');
+    addonAPI.window.loadURL(winId, 'file://' + __dirname + '/hello.html');
+    const options = {
+        type: 'info',
+        buttons: ['OK'],
+        title: 'TcResume',
+        message: 'Hello. Resume occurred!!'
+    };
+    addonAPI.window.showMessageBox(winId, options);
+}
+```
+
+You need to remove the installed addon, build a package with the HTML file and changed JavaScript file, and install the new package.
+
+If there's no issue, you will see the screen as below:
+
+![Resume with HTML and pop-up](media/resume_html_popup.png)
+
+### Return to main page of Web application
+
+In the implementation above, it is impossible to return to the main page of the Web application because it loads `hello.html` whenever the resume event happens.
+
+You have 2 issues to resolve the problem:
+
+- Showing the HTML page in every resume event
+- No way to return to the main page of the Web application
+
+1. Avoid showing the HTML page in every resume event
+
+   The reason of this issue is that you used `loadURL` in the callback of resume event which occurs in every resuming of the Web application.
+   You have to move the code to invoke the API to somewhere else. In the list of addon events, you can find `lcPrelaunch` which occurs only once when a Web application runs. This event gives you a chance to do something only once just after you launch a Web application.
+
+   To handle the event, you have to add the handler again. You can use any name of it if it is not used for other event. Following shows the case if the name is `myPrelaunch`:may remember that you have removed the code lines related to `lcPrelaunch` when you tried to run the pure addon template. It is time to revive it and you need to move the `loadURL` from `resume()` to `myPrelaunch()`.
```

Comment:
Here's my suggestion based on your one:

- Avoid showing the HTML page in every resume event

    The HTML page appears in every resume because of the `loadURL` in the callback of the resume event, which occurs when you resume the Web application. Therefore, you must remove the code from `resume(winId)`. The list of addon events, includes `lcPrelaunch`, which occurs only once when a Web application runs. It is recommended to move the `loadURL` to a callback for `lcPrelaunch`. The callback must have a unique name. The following code shows a callback function named `myPrelaunch`:

    ```
    activate() {
        ...
        this.evt.on('lcPrelaunch', this.myPrelaunch);
        ...
    } 
    deactivate() {
        ...
        this.evt.off('lcPrelaunch', this.myPrelaunch);
    }
    myPrelaunch(winId, orig) {
        addonAPI.window.loadURL(winId, 'file://' + __dirname + '/hello.html');  
    }
    ...
    ```

Please verify